tests/data/samples/sample_parameter_config.cfg

```diff
@@ -0,0 +1,135 @@
+#-- ========================================================================== --#
+#--                                                                            --#
+#--       This RVIC namelist contains options and paths for the                --#
+#--       development of the RVIC parameter file.                              --#
+#--                                                                            --#
+#--                                                                            --#
+#-- ========================================================================== --#
+
+# Note:  namelist is parsed by the python ConfigParser module.  %(Interpolation) is
+#        supported inside [sections] only.
+
+[OPTIONS]
+#-- ====================================== --#
+#--Level to log output at (char) --#
+# valid values: DEBUG, INFO, WARNING, ERROR, CRITICAL
+LOG_LEVEL: DEBUG
+
+#--Print output to console in addition to the log file (bool) --#
+# valid values: True, False
+VERBOSE: True
+
+#--Delete temporary files, only used if REMAP=True (bool) --#
+# valid values: True, False
+CLEAN: False
+
+#--case description (char)  --#
+CASEID: sample
+
+#--routing domain grid shortname (char)  --#
+GRIDID: COLUMBIA
+
+#--case run directory (char)  --#
+CASE_DIR: /tmp/%(CASEID)s
+
+#--Directory to use for temporary read/write operations (char)  --#
+TEMP_DIR: %(CASE_DIR)s/temp/
+
+#--Remap Unit Hydrographs from [ROUTING] grid to [DOMAIN] grid (bool)  --#
+# valid values: True, False
+REMAP: False
+
+#--Aggregate all [POUR_POINTS] inside each [DOMAIN] grid cell (bool)  --#
+# This should only be used when routing to coastal grid cells for CESM
+AGGREGATE: False
+
+#--Size of pad to add to aggregated files prior to remapping (int) --#
+AGG_PAD: 25
+
+#-- Output parameter file format (char) --#
+# Valid Values: NETCDF3_CLASSIC, NETCDF3_64BIT, NETCDF4_CLASSIC, and NETCDF4
+# For use with CESM, NETCDF3_CLASSIC is recommended.
+NETCDF_FORMAT: NETCDF4
+
+#-- Output parameter file compression options --#
+# Descriptions of these options can be found in
+NETCDF_ZLIB: False
+NETCDF_COMPLEVEL: 4
+NETCDF_SIGFIGS: None
+
+#-- Length of unit hydrograph subset in days (int) --#
+SUBSET_DAYS:
+
+#-- Constrain the final unit hydrographs sum to be less than or equal to the domain fractions --#
+# True when routing to coastal grid cells, else False
+CONSTRAIN_FRACTIONS: False
+
+SEARCH_FOR_CHANNEL: False
+
+#-- ====================================== --#
+
+[POUR_POINTS]
+#-- ====================================== --#
+#-- Path to Pour Points File (char) --#
+# A comma separated file of outlets to route to [lons, lats] - one coordinate pair per line (order not important)
+# May optionally include a column [names] - which will (if not aggregating) be included in param file
+FILE_NAME: tests/data/samples/sample_pour.txt
```

Comment:
Unlike `RVIC convolution` that uses a huge dataset, I think it is pretty reasonable to test `RVIC parameters` on local files. Good choice!